src/Redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+// Actions
+const GET_STATUS = 'GET_STATUS';
```

Comment:
Kinldy make the necessary changes so that the value of each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux)